src/identity-providers/add/DetailSettings.tsx

```diff
@@ -140,6 +159,35 @@ export const DetailSettings = () => {
   const isOIDC = id.includes("oidc");
   const isSAML = id.includes("saml");
 
+  const loader = async () => {
+    const loaderMappers = await adminClient.identityProviders.findMappers({
+      alias: id,
+    });
+
+    const loaderMapperTypes =
+      await adminClient.identityProviders.findMapperTypes({ alias: id });
+
+    const components = (loaderMappers as IdPWithMapperAttributes[])?.map(
```

Comment:
The `loaderMappers` variable already has a type associated with it, why does it need to be cast to another type?

Reply:
There should be no need to use an optional chain here as according to the types this is always an array.
```suggestion
    const components = (loaderMappers as IdPWithMapperAttributes[]).map(
```